archive.css

```diff
@@ -0,0 +1,20 @@
+#container{
+	display:flex;
+	display:-webkit-flex;
```

Comment:
One was the general one other was for the browsers which support webkit.
Now i had changed the code, i used bootstrap Grid loyout instead of
flexbox, so plese check if everytihng is alright.

On Fri, Aug 28, 2015 at 9:53 AM, Siddharth Kannan <notifications@github.com>
wrote:

> In archive.css
> <https://github.com/students-alumni-cell-iitkgp/aam-2016/pull/54#discussion_r38219998>
> :
>
> > @@ -0,0 +1,20 @@
> > +#container{
> > +	display:flex;
> > +	display:-webkit-flex;
>
> @hitesh4 <https://github.com/hitesh4>
>
> Why are there multiple display rules? You should use appropriate
> prefixes, if required.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/students-alumni-cell-iitkgp/aam-2016/pull/54/files#r38219998>
> .
>